### PR TITLE
Add option to specify the version code of an app

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

**Comment by mvdan** [comment]
`fdroidserver` uses `:` to separate appids and vercodes, like `foo.bar:123`. Use that here too for consistency.

File: cmd/fdroidcl/show.go [hunk 1]
```diff
@@ -47,11 +48,37 @@ func findApps(ids []string) []*fdroidcl.App {
 	apps := appsMap(mustLoadIndexes())
 	result := make([]*fdroidcl.App, len(ids))
 	for i, id := range ids {
-		app, e := apps[id]
-		if !e {
-			log.Fatalf("Could not find app with ID '%s'", id)
+		s := strings.Index(id, "=")
```
Comment (mvdan) [inline cmd/fdroidcl/show.go]: `s` is usually a name for strings, not integers.

File: cmd/fdroidcl/show.go [hunk 2]
```diff
@@ -47,11 +48,37 @@ func findApps(ids []string) []*fdroidcl.App {
...
+			if !found {
+				log.Fatalf("Could not find version %d for app with ID '%s'", ver, id)
+			}
+			result[i] = app
+		} else {
+			app, e := apps[id]
+			if !e {
+				log.Fatalf("Could not find app with ID '%s'", id)
+			}
+			result[i] = app
```
Comment (mvdan) [inline cmd/fdroidcl/show.go]: This bit of code is now duplicated. You should split the logic so that it isn't.

**Comment by jspricke** [comment]
Thanks for your comments, I've added commits for them. If you are ok with the patches, I will squash them.

**Comment by mvdan** [comment]
Looks good now. Squash and I'll merge.

**Comment by jspricke** [comment]
Squashed.

**Comment by mvdan** [comment]
The commit message is missing "Fixes #6" in a separate line.

**Comment by jspricke** [comment]
* Daniel Martí <notifications@github.com> [2016-08-11 08:14]:
> The commit message is missing "Fixes #6" in a separate line.

Oh, totally missed that one, added.

**Pull request merged**